src/core/entities/MRSkyBoxEntity.js

```diff
@@ -57,7 +57,8 @@ export class MRSkyBoxEntity extends MREntity {
      * @description Lifecycle method that is called when the entity is connected.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?